resources/build_targets.inc.sh

```diff
@@ -0,0 +1,43 @@
+#
+# Build a list of all targets that we will be building, in a specific order:
+# * release
+# * experimental
```

Comment:
Doesn't this also add to the list of external targets in build_external_targets.txt?

Reply:
I think so, yes.